tests/cpp/tests/test_array.cpp

```diff
@@ -95,7 +95,8 @@ TEST_CASE("ArrayIterator") {
         }
 
         WHEN("We create a mask over the array and create a masked iterator") {
-            auto mask = std::vector<double>{true, false, false, true, false, false, false, true, false};
+            auto mask =
```

Comment:
Please don't edit unrelated functions for formatting. It clutters the git history and without https://github.com/dwavesystems/dwave-optimization/issues/113 it's not even deterministic.